lms/djangoapps/notification_prefs/views.py

```diff
@@ -90,6 +90,25 @@ def decrypt(token):
         return UsernameCipher._remove_padding(decrypted)
 
 
+def enable_notifications(user):
+    """Enable notifications for a user"""
+    UserPreference.objects.get_or_create(
```

Comment:
Perhaps clarify in these doc strings that currently notifications are only used for forum digests.